flink-ai-flow/examples/wide_and_deep/README.md

```diff
@@ -0,0 +1,86 @@
+In the codes, we will show how to train a model with TF-on-Flink, which can directly consume training samples from flink and how to define and execute the streaming training and streaming prediction tasks with AIFlow.
+
+### Environment Requirements
+1. Flink-AI-Flow([ref](https://github.com/alibaba/flink-ai-extended/blob/master/flink-ai-flow/QUICKSTART.md))
+2. TF-on-Flink([ref](https://github.com/alibaba/flink-ai-extended/tree/master/deep-learning-on-flink))
+3. Flink(1.11+)
+4. Kafka(1.x or 2.x)
+5. Hadoop
+6. Zookeeper
+
+After building TF-on-Flink and installing Flink-AI-Flow, we need to put `.jar` files under the `flink-1.11.3/lib` dir.
+Find the  `flink-ml-tensorflow-0.4-SNAPSHOT-jar-with-dependencies.jar` in your TF-on-Flink project's target dir.
+Then copy the jar and the https://repo1.maven.org/maven2/org/apache/flink/flink-sql-connector-kafka_2.11/1.11.3/flink-sql-connector-kafka_2.11-1.11.3.jar into `flink-1.11.3/lib` dir. 
+
+
+### Usage
+#### HDFS Initialization
+
+- Edit `$HADOOP_HOME/etc/hadoop/core-site.xml` of hadoop:
+    ```shell
+    <property>
+    <name>fs.default.name</name>
+    <value>hdfs://127.0.0.1:9000</value>
+    </property>
+    ```
+- Start hadoop
+    ```shell
+    hadoop namenode -format
+    $HADOOP_HOME/sbin/start-all.sh
+    ```
+- Create directory in hdfs and upload codes
+    ```shell
+    hdfs dfs -mkdir /demo
+    hdfs dfs -put ./code.zip /demo
+    ```
+  Note, the `code.zip` must be produced by zipping the `wide_and_deep/dependencies/python/code` directory.
+  If users define their own logic in the python files under this dir, they should reproduce the zip file by running
+  ```shell
+  cd dependencies/python && zip -r code.zip code
+  ```
+  
+#### Kafka Initialization
+In the project, we will use Kafka to generate a data stream and the dataset used is `census_data/adult.data`. 
+To download the dataset, run `python wide_and_deep/dependencies/python/code/census_dataset.py` to save the dataset in `/tmp` dir.
+   
+Then, run `python wide_and_deep/dependencies/python/kafka_util/census_kafka_data.py` to create topics and generate kafka data stream.
+
+#### Start up the whole environment
+- Start flink:
+```shell
+start-cluster.sh
+```
+- Start AIFlow Server
+```shell
+python example_utils/example_server.py
+```
+Note, the user should refer to QuickStart for starting AIFlow correctly.
+
+#### Run the Wide and Deep project's workflow
+Before running the demo, the user need to check the 2 project.yaml is configured properly:
+
+```shell
+1. flink-ai-flow/examples/wide_and_deep/project.yaml
+2. flink-ai-flow/examples/wide_and_deep/dependencies/python/code/project.yaml
+```
+Make sure these 2 files use same config.
+
+Only for the first time of running the project, we need to init the AIFlow:
+```shell
+python wide_and_deep/workflows/wdl/census_init.py 
+```
+
+Then, run the workflow:
+```shell
+python workflows/wdl/wdl.py
```

Comment:
we should ask users to run the workflow in the same dir:
Maybe `python workflows/wdl/wdl.py` -> `python wide_and_deep/workflows/wdl/wdl.py`